msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -106,7 +103,7 @@ GET /v1/meetings/{meetingId}/participants/{participantId}?tenantId={tenantId}
 
 ---
 
-The JSON response body for `GetParticipant` API is:
+The following JSON response body for `GetParticipant` API is:
```

Comment:
Please remove 'following' here.

Reply:
Hi @AkJo, 
The word following is suggested in MSTP, if any information is followed by the sentence ending in :. Could you please let me know the reason, as it is grammatically correct and used through Ms Teams doc site. the word 'following' and 'preceding' are suggested by MSTP.